index.js

```diff
@@ -384,7 +384,7 @@ function signApplicationAsync (opts) {
         })
           .then(function () {
             debuglog('Signing... ' + opts.app)
-            execFileAsync('codesign', args.concat('--entitlements', opts.entitlements, opts.app))
```

Comment:
I am not sure, but my understanding back then is that: If a promise is returned from a function, that function `.then()` only works after the promise's fulfilled. However, if nothing's returned from a function yet a promise is created, that function `.then()` starts immediately instead of waiting for that promise to be fulfilled... Not sure if this is an intelligible saying. 😖
Anyway... It works with the `return` lol.